setup.py

```diff
@@ -47,11 +47,6 @@ def load_readme():
             "pytest-cov",  # Code coverage
             "jupyter",  # For runnable examples
         ],
-        "lint": [
-            "black==23.3.0",
```

Comment:
where are these defined for people not using pre-commit now?

Reply:
We could keep both, but I like the one-thing-one-place principle. To keep both, it would be nice if there would be an automatic way to keep config in sync

Reply:
So do I, if possible. Your example posted below will I fine, I guess :-)

```
python -m pip install pre-commit
pre-commit run --all-files
```